extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -382,6 +384,34 @@ export class ColumnsTable extends ModelViewBase implements IDataComponent<Predic
 		return warningButton;
 	}
 
+	public async openDialog(title: string, dialogName?: string, width?: azdata.window.DialogWidth, dialogStyle?: azdata.window.DialogStyle, dialogPosition?: azdata.window.DialogPosition) {
+		if (!this._isOpen) {
```

Comment:
I think we chatted offline, we shouldn't need this flag, also, I don't see this being set to false. so the dialog can really be opened once according the code here.

Reply:
Correct. I have made changes and found that it works just fine without the flag.